config/locales/views/en.offline_cancellation_mailer.yml

```diff
@@ -0,0 +1,21 @@
+en:
+  views:
+    offline_cancellation_mailer:
+      send_notification:
+        subject: Your reservation for %{product} has been canceled
+        body:
+          html: |
+            %{product}, the instrument you
+            [have reserved for %{reservation_time}](%{order_detail_url}), is down.
+            Your reservation has been canceled at no cost.
+
+            Please contact [%{facility_name}](%{facility_url})
```

Comment:
It may be safe to change this to "Please contact the `facility_name`" given the values I see in NU's database but can we count on that?